src/llscan.cc

```diff
@@ -668,6 +672,59 @@ char** FindReferencesCmd::ParseScanOptions(char** cmd, ScanType* type) {
   return &cmd[optind - 1];
 }
 
+// Walk all contexts previously stored and print search_value_
+// reference if it exists. Not all values are associated with
+// a context object. It seems that Function-Local variables are
+// stored in the stack, and when some nested closure references
+// it is allocated in a Context object.
+void FindReferencesCmd::ReferenceScanner::PrintContextRefs(
+    SBCommandReturnObject& result, Error& err) {
+  ContextVector* contexts = llscan_->GetContexts();
+  v8::LLV8* v8 = llscan_->v8();
+
+  ContextVector visited_contexts;
+  for (auto ctx : *contexts) {
+    if (std::find(visited_contexts.begin(), visited_contexts.end(), ctx) !=
+        visited_contexts.end())
```

Comment:
Why do we need to keep track of visited contexts? Can we have duplicates on LLScan::contexts_?